### ISSUE TITLE
abortBroadcast causes a crash all the time at the first installation

### ISSUE THREAD

**Issue by fabriziomoscon** [evollu/react-native-fcm#215]
Please include following information for better support

RN 0.38
react-native-fcm 2.3.2
Android 5.1.1 OnePlus A0001
running in foreground
 log
```
D/InstanceIdService: Refreshed token: erJHmDV3GFg:APA91bElAymZXwwYb0QH2wvEQgbI75fQNlKcAoHbBbi0fP9yLRvAdCl5EDMdCEnwjnr4dYSQq23NdI3aCNEgdsTgSOwYqNxD5U8z3wIznJZInvdkdafk9zffnCRqEJyNiyTChgxObYLR
E/BroadcastReceiver: BroadcastReceiver trying to return result during a non-ordered broadcast
      java.lang.RuntimeException: BroadcastReceiver trying to return result during a non-ordered broadcast
      at android.content.BroadcastReceiver.checkSynchronousHint(BroadcastReceiver.java:797)
      at android.content.BroadcastReceiver.abortBroadcast(BroadcastReceiver.java:692)
      at com.evollu.react.fcm.FIRMessagingModule$1.onReceive(FIRMessagingModule.java:131)
      at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:867)
      at android.os.Handler.handleCallback(Handler.java:739)
      at android.os.Handler.dispatchMessage(Handler.java:95)
      at android.os.Looper.loop(Looper.java:135)
      at android.app.ActivityThread.main(ActivityThread.java:5292)
      at java.lang.reflect.Method.invoke(Native Method)
      at java.lang.reflect.Method.invoke(Method.java:372)
      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
W/GCM: Mismatched messenger
```

Comparing the `react-native-push-notification` implementation for the same feature I don't see `abortBroadcast`
https://github.com/zo0r/react-native-push-notification/blob/263a63e6ec4b1e417cf1ce5775994c66d942e48d/android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java#L70-L83

Is there a reason to try to abort the broadcast?

**Comment by evollu** [comment]
hmm. will remove it

**Comment by evollu** [comment]
published in 2.4.3. can you verify if issue is fixed?

**Comment by fabriziomoscon** [comment]
hey thanks! The app doesn't crash anymore.
I can't tell if there will be other effects removing the abortBroadcast calls, but for what I can see my issues is resolved
